src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { APIGatewayProxyResult, Handler } from 'aws-lambda';
+import { UserController } from './core/UserController';
+import { UserDataAccess } from './core/UserDataAccess';
+import { ExceptionMapper } from './exception/ExceptionMapper';
+import { User } from './model/User';
+import { UserCreate } from './model/UserCreate';
+import { UserUpdate } from './model/UserUpdate';
+
+const userController: UserController = new UserController(new UserDataAccess());
+const exceptionMapper: ExceptionMapper = new ExceptionMapper();
+
+export const getUser: Handler = async (event: any): Promise<APIGatewayProxyResult> => {
+  return await handle(async () => {
+    const id: string = event.pathParameters.id;
+    const user: User = await userController.getUser(id);
+    return {
+      statusCode: 200,
+      body: JSON.stringify(user),
+    };
+  }, event);
+};
+
+export const createUser: Handler = async (event: any): Promise<APIGatewayProxyResult> => {
+  return await handle(async () => {
```

Comment:
> Have you in the past seen issues / limitations with handling exceptions via a wrapper rather than having them explicitly defined in the handler code?

@afaq-sf   I don't believe so.  Or, at least, the value gained in not repeating the wrapper code has always outweighed the edge cases?  I could imagine a very specific error which would like to be handled only by the method which called it.  I think I would consider that more a code smell than an argument for handing errors in the handler.  Although there are always exceptions. 

All rest frameworks I've used provide default error handlers (wrappers) like this and allow you to create a mapping for any types of errors you don't wish to be handled by the default.  This adds great value knowing I can't miss an error and all errors will have a predictable result to the client.  But I can still override that behavior if I need to.  

Since I'm not using a framework here I created a simple wrapper.  However at the onset of a project I would consider whether it may be valuable to use a framework instead of raw handlers.  There are lambda-express plugins.  Similar with fastify and nestjs I believe.  These could offer real value in error handling, validation, middleware, dependency injection etc.